### PR TITLE
v1.1.0

### PR DESCRIPTION
**Pull request by kisnikser**
This PR introduces many changes:
1. Rename `StraightThroughBernoulli` into `StochasticTimesSmooth`
2. Implement truly `StraightThroughBernoulli`
3. Conduct demo experiment with new `StraightThroughBernoulli` and update it for `StochasticTimesSmooth`
4. Change tricky notation in some distributions into `logits` and `probs`
5. Refactor old and add new tests
6. Refactor documentation and add custom `relaxit.css` style
7. Update `README.md` in demo and root directories to match new distribution names

### CODE REVIEW

**Review by DorinDaniil — COMMENTED** [review]
Good job, add new methods

**Review by DorinDaniil — APPROVED** [review]
Good job, add new methods
